main.go

```diff
@@ -41,14 +42,25 @@ var readProxyType = func() string {
 	} else {
 		prompt := &survey.Select{
 			Message: "Choose proxy type:",
-			// TODO Refactor types to Enums
 			Options: proxyTypes,
 		}
 		survey.AskOne(prompt, &proxyType)
 	}
-	return proxyType
+	return ProxyType(proxyType)
 }
 
+// ProxyType is one of Pod, CloudSQL, VM
+type ProxyType string
+
+const (
+	// ProxyTypePod Pod proxy type
+	ProxyTypePod ProxyType = "POD"
+	// ProxyTypeSQL CloudSQL proxy type
+	ProxyTypeSQL ProxyType = "CLOUD_SQL"
+	// ProxyTypeVM VM proxy type
+	ProxyTypeVM ProxyType = "VM"
```

Comment:
Changelog says that proxy type VM was removed.

Reply:
From the user interface, but I still kept it in code :D ok, will remove completely.. and maybe add later on.

Reply:
Removed and ready for review here https://github.com/AckeeCZ/goproxie/pull/29